spec/features/unsubscribe_all_spec.rb

```diff
@@ -0,0 +1,69 @@
+RSpec.feature "Bulk unsubscribe after receiving confirmation link" do
+  include GdsApi::TestHelpers::EmailAlertApi
+  include TokenHelper
+
+  scenario do
+    given_i_have_multiple_subscriptions
+    when_i_visit_the_manage_my_subscriptions_page
+    and_i_click_on_unsubscribe_from_everything
+    and_i_confirm_to_unsubscribe
+    then_i_can_see_i_have_been_unsubscribed
+  end
+
+  def given_i_have_multiple_subscriptions
+    @address = "test@test.com"
+    @subscriber_id = 123
+    stub_email_alert_api_has_subscriber_subscriptions(
+      @subscriber_id,
+      @address,
+      nil,
+      subscriptions: [subscription, subscription],
+    ).times(3)
+     .then
+     .to_return(status: 200, body: subscriber_with_no_subscriptions_response)
```

Comment:
Since the `times(3)` is a bit tricky to understand, can we simplify to just return the original response each time? I understand it's not as realistic, but I don't think anyone will notice.

Reply:
Sure, I've removed it 👍